swarm/network/discovery_test.go

```diff
@@ -58,3 +68,161 @@ func TestDiscovery(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+const (
```

Comment:
minor: As these constants are used only in one function per each, I see no need to define them on package level polluting the scope.

Reply:
no they are used in `TestInitialPeersMsg` and `testInitialPeersMsg`, they belong together and they constant-like ie, dont want to pollute the arguments with them.